### ISSUE TITLE
Accessibility : Stats Posting Activity Improvements

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10975]
### Expected behavior

You should be able to select each month and navigate through the different days to hear an overview of the activity.

### Actual behavior
The entire component is selected but you aren't able to get any details of what's within it. 

<img src="https://user-images.githubusercontent.com/1509205/71025576-4f5f7400-20d5-11ea-884c-cff644b7b6d6.png" width="320">

### Steps to reproduce the behavior
1. Go to my site. 
2. Select Stats 
3. Select the "Posting Activity" section
